File: Homework_OOP_2/Stork.java
package Homework_OOP_2;

public class Stork extends Bird implements BirdFly{

    public Stork(int height, int weight, String colorEye, int hightFly) {
        super(height, weight, colorEye, hightFly);
    }
    
    @Override
    public void toFly() {
        System.out.printf("Я летаю на высоте %d метров", super.getHeightFly());
    }

    @Override
    public void animalSound() {
        System.out.println("Курлыкает");
        
    }

    @Override
    public String toString() {
        return String.format("Аист: %s", super.toString());
    }
}

File: Homework_OOP_7/classes/bird/stork.py
from ..interfeces.BirdFly import BirdFly
from .bird import Bird

class Stork(Bird, BirdFly):
    def __init__(self, name, height, weight, colorEye, fly):
        super().__init__(name, height, weight, colorEye, fly)

    def animalSay(self):
        print(f"{super().animalSay()} 'Курлы-курлы' ")

    def birdFly(self):
        BirdFly.birdFly(self)

    def __str__(self) -> str:
        return super().__str__() 